### DISCUSSION TITLE
Problems with solver on MLCP problems

### DISCUSSION THREAD

**Discussion by ElMagnifico21** [pysal/spopt#383] [Q&A]
Hello everyone, I wanted to ask how to solve a problem with the solver library, since when running the code in the part of the solutions with solver these throw me an error, it is worth mentioning that I am running the code in the jupyter platform!

That's why I wanted to ask if there was any other option to run the code perhaps as another library as gurobi?

**Comment by jGaboardi** [comment]
@ElMagnifico21 thanks for using `spopt`. If you are hitting an error, please open an issue. In order to assist in a solution for your problem at minimum we'll need:
* the full traceback of your error (copy and paste, not a screenshot)
* a minimum reproducible example that can be run to replicate your error (again, copy and paste, not screen shot)
* the version of `spopt` you are running along with other major dependencies (`pulp`, `geopandas`, etc.) including Python
* the solvers you have installed (run ` import pulp;  pulp.list_solvers(onlyAvailable=True)`)
